modules/apps/document-library/document-library-web/src/main/java/com/liferay/document/library/web/internal/display/context/CopyDLObjectsDisplayContext.java

```diff
@@ -163,23 +153,19 @@ public void setViewAttributes() {
 		}
 	}
 
-	private FolderItemSelectorCriterion _getFolderItemSelectorCriterion(
-		long folderId, long repositoryId) {
+	private long _getFolderId() {
+		if (_dlObjectIds.length > 1) {
```

Comment:
When we use `DLFolderConstants.DEFAULT_PARENT_FOLDER_ID` as default value we are not allowing to copy a file on the Home folder.